components/phpunit_bridge.rst

```diff
@@ -119,7 +119,25 @@ an arbitrary value (ex: ``320``) will make the tests fails only if a higher
 number of deprecation notices is reached (``0`` is the default value). You can
 also set the value ``"weak"`` which will make the bridge ignore any deprecation
 notices. This is useful to projects that must use deprecated interfaces for
-backward compatibility reasons.
+backward compatibility reasons, or for libraries with lots of vendors that
+can create a sudden backlog of deprecations to fix that would otherwise prevent
+unrelated pull requests from being merged… which leads us to ``"weak_vendors"``.
```

Comment:
Replaced the ellipsis with `described below` so that the reader doesn't feel like they missed something.